extensions/prototype-content-with-issues-table/app.R

```diff
@@ -4,10 +4,11 @@ library(DT)
 library(connectapi)
 library(dplyr)
 library(purrr)
+library(tidyr)
```

Comment:
This is add for `replace_na()` yeah? Are there other functions we use `tidyr` for?

Reply:
Yes, added in response to [this suggestion](https://github.com/posit-dev/connect-extensions/pull/33#discussion_r1989918296):

> Would [replace_na()](https://tidyr.tidyverse.org/reference/replace_na.html) do what you're looking for here? 

Only used within datatable render on L121 `mutate(content_title = replace_na(content_title, ""))`.

Reply:
Got it, that's fine but something for us to remember if that ends up getting factored differently, we can clean this up.